pywin32_postinstall.py

```diff
@@ -215,11 +214,17 @@ def RegisterPythonwin(register=True):
     ## Installer executable doesn't seem to pass anything to postinstall script indicating if it's a debug build,
     pythonwin_exe = os.path.join(lib_dir, "Pythonwin", "Pythonwin.exe")
     pythonwin_edit_command=pythonwin_exe + ' -edit "%1"'
+    pythonwin_new_command=pythonwin_exe + ' -new "%1"'
+    pythonwin_run_command=pythonwin_exe + ' -run "%1"'
 
     keys_vals = [
         ('Software\\Microsoft\\Windows\\CurrentVersion\\App Paths\\Pythonwin.exe', '', pythonwin_exe),
         ('Software\\Classes\\Python.File\\shell\\Edit with Pythonwin', 'command', pythonwin_edit_command),
         ('Software\\Classes\\Python.NoConFile\\shell\\Edit with Pythonwin', 'command', pythonwin_edit_command),
+        ('Software\\Classes\\Python.File\\shell\\Edit in new Pythonwin', 'command', pythonwin_new_command),
```

Comment:
I'm not sure these are worthwhile - the more options, the harder it is to find the ones you actually want. And once you have multiple pythonwin's, you've kinda broken the "Edit with Pythonwin" (as you can't control which one). IOW, I'm not sure we should *encourage* multiple pythonwins?

Reply:
I can remove the 2 context commands again from 2nd commit. My motivation was:
* The -run thing lets you comfortably run scripts and inspect / debug the aftermath in PythonWin  (with the extended Debugger etc. to come)
* the -new : I indeed use a 2nd or even 3rd PythonWin quite often as its fast and light-weigth (Actually I use even more context commands for different PY versions).  The -new option is also rather unkown otherwise and one is forced into one instance. Even fatter editors like VSCode etc. can easily have more instances. The distinction is still there: The "Edit with Python" always goes into the one "main" instance. While the secondary -new ones don't listen on DDE at all.
* It may be more easy to remove the commands for a programmer than to know about the alternative startup.